### PR TITLE
WICS-0: Message IDs

### PR DESCRIPTION
**Pull request by ntsvetko**
I'm referencing https://trello.com/c/wNOjfIOH/8-wics-0-message-ids-demo#.

I have added message IDs, so now you can look up a message by its ID.

### CODE REVIEW

File: message_board.py [hunk 1]
```diff
@@ -23,12 +30,17 @@ def get_message_by_id(self, id):
     print('Enter \"list\" to see all messages.')
     print('Enter \"post\" to post a message.')
     print('Enter \"exit\" to destroy me.\n')
```
Comment (florenciacomuzzi) [inline message_board.py:32]: I noticed there's a newline here. Do we want an extra line here?

File: message_board.py [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
         return self.messages
 
     def post_message(self, msg):
-        self.messages.append(msg)
+        self.messages[self.get_unique_id()] = msg
 
     def get_message_by_id(self, id):
         pass
 
+    def get_unique_id(self):
```
Comment (florenciacomuzzi) [inline message_board.py:20]: Can we add some comments here?

**Review by florenciacomuzzi — APPROVED** [review]
LGTM

File: message_board.py (hunk 2 above)
Comment (ntsvetko) [reply message_board.py:20]: yeah!

**Pull request merged**